solutions/api/src/specs/public-api.yaml

```diff
@@ -64,63 +64,76 @@ paths:
           description: Redirect the user agent to the redirect uri
           headers:
             Location:
-              description: Redirect URI with `code` and `state` on successful journey completion, or `error` and `error_description` parameters if authorization fails.
+              description: Redirect URI with `code` and `state` on successful journey completion, or `error`, `error_description` and `state` parameters if authorization fails.
               schema:
                 format: uri
                 type: string
               examples:
                 success:
                   value: https://signin.account.gov.uk/callbacks/account-deletion?code=anAuthCode&state=aRandomString
                 user_aborted:
-                  value: https://signin.account.gov.uk/callbacks/account-deletion?error=access_denied&error_description=E1001
+                  value: https://signin.account.gov.uk/callbacks/account-deletion?error_description=access_denied&error=E1001&state=aRandomString
```

Comment:
`error` should be `access_denied` and `error_description` should be the Exxx number.  This applies to all below aswell